### PR TITLE
Design story backend+frontend

### PR DESCRIPTION
**Pull request by rounak610**

### CODE REVIEW

File: app/services/s3_providers/s3_service.go [hunk 1]
```diff
@@ -137,6 +137,10 @@ func NewS3Service() (*S3Service, error) {
 	awsBucketName := config.AWSBucketName()
 	awsRegion := config.AWSRegion()
 
+	fmt.Println("awsAccessKeyID:", awsAccessKeyID)
```
Comment (sasikanthlanka) [inline app/services/s3_providers/s3_service.go:140]: Remove logging of credentials

**Review by sasikanthlanka — CHANGES_REQUESTED** [review]

File: executor.go [hunk 2]
```diff
@@ -269,6 +294,14 @@ func main() {
 		log.Fatalf("Error providing OpenAiClient: %v", err)
 	}
 
+	// Provide ClaudeClient
+	if err = c.Provide(func() *llms.ClaudeClient {
+		apiKey := config.ClaudeAPIKey()
+		log.Println("Claude API Key:", apiKey)
```
Comment (sasikanthlanka) [inline executor.go:300]: Remove api key logging, also remove creating singleton claude client altogether. Create new instance with customer key where ever possible.

**Review by sasikanthlanka — CHANGES_REQUESTED** [review]

File: app/services/s3_providers/s3_service.go (hunk 1 above)
Comment (rounak610) [reply app/services/s3_providers/s3_service.go]: removed

File: executor.go (hunk 2 above)
Comment (rounak610) [reply executor.go]: removed

**Pull request closed without merging**